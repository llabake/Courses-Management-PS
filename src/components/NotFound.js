import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <div>
    <div className="jumbotron">
      <h3 className="text-center">This is embarrassing, something is missing......</h3>
      <p className="text-center"><Link to="/">Home</Link></p>
    </div>
  </div>
);

export default NotFound;
