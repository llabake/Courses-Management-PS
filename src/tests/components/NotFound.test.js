import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../../components/NotFound';

describe('<NotFound />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('#text').text()).toMatch('This is embarrassing, something is missing......');
  });
});
