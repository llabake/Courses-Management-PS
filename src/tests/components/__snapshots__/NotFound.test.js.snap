// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render correctly 1`] = `
<div>
  <div
    className="jumbotron"
  >
    <h3
      className="text-center"
      id="text"
    >
      This is embarrassing, something is missing......
    </h3>
    <p
      className="text-center"
    >
      <Link
        replace={false}
        to="/"
      >
        Home
      </Link>
    </p>
  </div>
</div>
`;
